api/src/main/java/org/openmrs/module/appointments/dao/impl/AppointmentDaoImpl.java

```diff
@@ -61,6 +52,30 @@ public List<Appointment> getAllAppointmentsReminder(String hours) {
         return criteria.list();
     }
 
+    @Override
+    public List<Appointment> getAllAppointments(Date forDate, AppointmentStatus status) {
```

Comment:
Let's add javadoc to describe what the method does.

Reply:
We can mark this as resolved.